lib/src/content/components/paginating_group/wolt_modal_sheet_page_transition_state.dart

```diff
@@ -60,17 +60,25 @@ enum WoltModalSheetPageTransitionState {
     }
   }
 
+  /// Returns an animation for the main content slide position based on the
+  /// provided parameters, supporting both LTR and RTL directions.
   Animation<Offset> mainContentSlidePosition(
     AnimationController controller,
     WoltModalSheetPaginationAnimationStyle style, {
     required double sheetWidth,
     required double screenWidth,
     required bool isForwardMove,
+    required bool isLTR, // New parameter to determine the text direction
   }) {
+    // Determine the direction multiplier based on isLTR and isForwardMove.
+    final directionMultiplier = (isLTR ? 1 : -1) * (isForwardMove ? 1 : -1);
```

Comment:
```suggestion
    final directionMultiplier = (isLTR ? 1 : -1) * (isForwardMove ? 1 : -1);
```

Then check if it is LTR here.

Reply:
done 4982dcf65a627145e5f3a59e13dd56aba6e424d4